tsconfig.json

```diff
@@ -111,6 +111,15 @@
       ],
       "@theia/getting-started/lib/*": [
         "packages/getting-started/src/*"
+      ],
```

Comment:
hello, why do we need this as part of this PR ?

Reply:
because navigation does not work, one cannot see that file search is used in plugin ext